WordPress/src/main/java/org/wordpress/android/ui/posts/PostConflictResolver.kt

```diff
@@ -84,6 +84,10 @@ class PostConflictResolver(
         return !isFetchingConflictedPost && PostUtils.isPostInConflictWithRemote(post)
     }
 
+    fun hasUnhandledAutoSave(post: PostModel): Boolean {
+        return PostUtils.isPostInConflictWithAutoSave(post)
```

Comment:
As per my explanation above, it'd be nice to rename `PostUtils.isPostInConflictWithAutoSave(post)` to something like `PostUtils.hasUnpublishedRevision(post)` or `PostUtils.hasRemoteAutoSave(post)` or similar. Wdyt?